scripts/write-symbols.js

```diff
@@ -0,0 +1,144 @@
+const { spawnSync } = require('child_process');
```

Comment:
add `use strict`

Reply:
Addressed in https://github.com/nodejs/node-api-headers/pull/7/commits/b9a57c3721c8be26c76a5e2777e05ed3886a938b